source/index.erb

```diff
@@ -2,36 +2,53 @@
 title: Energy Globe Award
 ---
 
-<div id="application">
-
-  <header>
-    <h1 class="headline">Energy Globe Award</h1>
-  </header>
-
-  <section ng-controller="AwardsController">
-
-    <section class="filters">
-      <h2>Kategorien</h2>
-      <ul class="category-filter" ng-repeat="category in filters.category.names">
-        <li>
-          <input type="checkbox" id="category_checkbox_{{category}}" ng-model="filters.category.filterMap[category]">
-          <label for="category_checkbox_{{category}}">{{category}}</label>
-        </li>
-      </ul>
-
-      <h2>Jahre</h2>
-      <ul class="year-filter" ng-repeat="year in filters.year.names">
-        <li>
-          <input type="checkbox" id="year_checkbox_{{year}}" ng-model="filters.year.filterMap[year]">
-          <label for="year_checkbox_{{year}}">{{year}}</label>
-        </li>
-      <ul>
-    </section>
-
-    <div class="award" ng-repeat="award in filteredAwards">
-      <h3>{{ award.title }}</h3>
+<header class="page-header">
+  <div class="page-header-container">
+    <div class="page-header-content">
+      <h1 class="page-header-headline">Energy Globe Award</h1>
```

Comment:
Hmm… I've got this style actuilly from @pferdefleisch, who's got a book on this. Repeating the `page-header` might not look nice, but it gives you 2 main advantages:

1. It's possible to refer to them without nesting. While nesting is a nice feature of SASS, you'll end up with several levels of nesting over time, which makes it hard to read.
2. If someone ever defines a `.container` somewhere else and writes a style `.container { … }` it will also affect this container accidentally. Repeating the "namespace" on every class prevents that.